selfdrive/car/honda/values.py

```diff
@@ -695,6 +699,7 @@ class CAR:
   CAR.CRV_EU: dbc_dict('honda_crv_executive_2016_can_generated', 'acura_ilx_2016_nidec'),
   CAR.CRV_HYBRID: dbc_dict('honda_crv_hybrid_2019_can_generated', None),
   CAR.FIT: dbc_dict('honda_fit_ex_2018_can_generated', 'acura_ilx_2016_nidec'),
+  CAR.HRV: dbc_dict('honda_fit_ex_2018_can_generated', 'acura_ilx_2016_nidec'),
```

Comment:
You will want an HRV specific DBC for this. Pull requests can be made to the opendbc repo, https://github.com/commaai/opendbc